src/lib.rs

```diff
@@ -420,17 +420,17 @@ macro_rules! __bitflags {
 #[cfg(bitflags_const_fn)]
 macro_rules! __fn_bitflags {
     (
-        $(#[$filtered:meta])*
+        $(#[$attr:ident $($attr_args:tt)*])*
```

Comment:
These could be simplified to `$(# $attrs:tt)*`, since the macro doesn't care about internal structure and only needs to propagate tokens.